elements/rh-badge/rh-badge.css

```diff
@@ -0,0 +1,38 @@
+/* stylelint-disable color-hex-length */
+
+:host {
+  border-radius: var(--pf-c-badge--BorderRadius, var(--pf-global--BorderRadius--lg, 180em));
```

Comment:
You'll want to update these to the rhds variables / default tokens https://red-hat-design-tokens.netlify.app/

Reply:
For example, for 

`min-width: var(--pf-c-badge--MinWidth, var(--pf-global--spacer--xl, 2rem));`

You would want to use 

`min-width: var(--rh-badge-MinWidth, var(--rh-space-2xl, 32px));`

Reply:
better `--rh-badge-min-width` per [conventions](https://github.com/RedHat-UX/red-hat-design-system/wiki/CSS-Styles#tokens-and-naming-conventions)

even better don't expose min-width as a token at all, unless specifically needed, users could just set `min-width` themselves